src/configuration/converters.ts

```diff
@@ -156,6 +157,20 @@ export function mapToScooterFaqs(scooterFaqs?: any) {
     .filter(isDefined);
 }
 
+export function mapToBonusProducts(bonusProducts?: any) {
```

Comment:
Should the imported BonusProduct type be set instead of any? And if its supposed to only be an array, it could be BonusProduct[], and then make sure its atleast used with an empty array where the function is used. then you dont need the isArray check